architecture/additional_concepts/authorization.adoc

```diff
@@ -238,6 +238,8 @@ containers].
 . The SELinux context of the container.
 . The user ID.
 . The use of host namespaces and networking.
+. Setting an FSGroup used for shared file systems
```

Comment:
FSGroups are used with unshared filesystems.  Instead, this should read:

`Allocating an FSGroup that owns the pod's volumes`